Makefile

```diff
@@ -19,27 +19,6 @@ generate-version-file: ## Generates the app version file
 test: generate-version-file ## Run tests
 	./scripts/run_tests.sh
 
-.PHONY: freeze-requirements
-freeze-requirements:
-	rm -rf venv-freeze
-	virtualenv -p python3 venv-freeze
-	# Make sure we are using the latest pip prior to requirements installation.
-	$$(pwd)/venv-freeze/bin/python3 -m pip install --upgrade pip
-	$$(pwd)/venv-freeze/bin/pip install -r requirements-app.txt --no-cache-dir
-	echo '# pyup: ignore file' > requirements.txt
-	echo '# This file is autogenerated. Do not edit it manually.' >> requirements.txt
-	cat requirements-app.txt >> requirements.txt
-	echo '' >> requirements.txt
-	$$(pwd)/venv-freeze/bin/pip freeze -r <(sed '/^--/d' requirements-app.txt) | sed -n '/The following requirements were added by pip freeze/,$$p' >> requirements.txt
-	rm -rf venv-freeze
-
-.PHONY: test-requirements
-test-requirements:
-	@diff requirements-app.txt requirements.txt | grep '<' \
-	    && { echo "requirements.txt doesn't match requirements-app.txt."; \
-	         echo "Run 'make freeze-requirements' to update."; exit 1; } \
-|| { echo "requirements.txt is up to date"; exit 0; }
-
```

Comment:
This makes sense, I hadn't considered the aspect of carrying out common tasks without knowing the underlying tooling as a point of convenience. I will restore this targets and update them.

Reply:
Thanks! we could test if we could move everything to poetry afterward, to see if we can make targets in there and rely on Python instead of bash. Just throwing out the idea for the future if we like this idea.

Reply:
Restored, as well as in `admin` and `DDL-API`.